src/middleware/etag/index.ts

```diff
@@ -5,30 +5,52 @@ type ETagOptions = {
   weak: boolean
 }
 
+/**
+ * List of headers to pass through on 304 responses. From the spec:
+ * > The response must not contain a body and must include the headers that
+ * > would have been sent in an equivalent 200 OK response: Cache-Control,
+ * > Content-Location, Date, ETag, Expires, and Vary.
+ */
+const RETAINED_304_HEADERS = [
+  'cache-control', 'content-location', 'date', 'etag', 'expires', 'vary'
+]
```

Comment:
I can imagine a scenario where an app wants to include additional headers. Should this be configurable via an option alongside `weak`?

Reply:
Yes, I also think it would be better for these to be options.